tool/pcore/src/main/java/org/openscience/cdk/pharmacophore/PharmacophoreMatcher.java

```diff
@@ -426,8 +426,8 @@ private IAtomContainer getPharmacophoreMolecule(IAtomContainer input) throws CDK
             for (IBond bond : pharmacophoreQuery.bonds()) {
                 if (!(bond instanceof PharmacophoreQueryAngleBond)) continue;
 
-                IAtom startQAtom = bond.getBegin();
-                IAtom middleQAtom = bond.getEnd();
+                IAtom startQAtom = bond.getAtom(0);
+                IAtom middleQAtom = bond.getAtom(1);
```

Comment:
Oh, duh... sorry, was distracted too much by the changes that I did not notice the getAtom(2)... not now, not when I reviewed your getBegin/getEnd patch...